src/main/webapp/index.html

```diff
@@ -46,32 +47,30 @@ <h2 class="panel-row__header u-align-with-first-panel u-text-shadow">Services (a
             </a>
             <h3 class="panel__section-heading u-vertical-center">Mail</h3>
           </div>
-          <p class="panel__section-subheading">From the last 7 days:
+          <p class="panel__section-subheading">From the last <span id="gmailNDays"></span> days:
           <div class="panel__content">
             <div class="panel__content-entry">
               <div class="panel__rounded-number u-background-mail">
-                <p class="panel__text-data-point u-text-white">35
+                <p class="panel__text-data-point u-text-white" id="gmailUnreadEmailsDays">
               </div>
               <p class="panel__text-data-point u-vertical-center">Unread Emails:
             </div>
             <div class="panel__content-entry">
               <img class="panel__content-icon" alt="clock-icon" src="assets/clock.png">
-              <p class="panel__text-large u-vertical-center">2 from last 3 hours
+              <p class="panel__text-large u-vertical-center"><span id="gmailUnreadEmailsHours"></span>&nbsp;from the last&nbsp;<span id="gmailMHours"></span>&nbsp;hours
```

Comment:
I could be wrong but it seems like `&nbsp` is used more for styling purposes. Is there a way to abstract this into CSS?

Reply:
`&nbsp` has a unique meaning in HTML, and cannot be abstracted. They should, however, be avoided, but this was a required workaround based on a style mistake I made in my UI pull request. This is corrected in the next dependent PR, #87 .